src/pykeen/evaluation/evaluator.py

```diff
@@ -267,11 +269,14 @@ def _param_size_search(
             values_dict[key] = start_value
             values_dict['slice_size'] = None
         elif key == 'slice_size':
+            if start_value is None:
+                raise ValueError('must specify a start value with key="slice_size"')  # TODO @lvermue check
```

Comment:
```suggestion
                start_value=ceil(model.num_entities / 2)
```

Reply:
thank youuuu. Let me know if there's anything else you want changed before merging this one